react/src/pages/InfoPage.js

```diff
@@ -214,7 +223,7 @@ const InfoPage = () => {
             </>:<>
                 <b onClick={() => initiateAppReload()}data-text={"Click here to reload the Foursight app."} className={"tool-tip"} style={{float:"right",cursor:"pointer"}}>{Char.Refresh}</b>
             </>}
-            <div className="tool-tip" data-text="Click to clear any caches." style={{float:"right",marginTop:"-1px",marginRight:"4pt",cursor:"pointer"}}>&nbsp;&nbsp;<img src={Image.ClearCache()} height="19" onClick={clearCache}/></div>
+            <div className="tool-tip" data-text="Click to clear any caches." style={{float:"right",marginTop:"-1px",marginRight:"4pt",cursor:"pointer"}}>&nbsp;&nbsp;<img alt="clearcache" src={Image.ClearCache()} height="19" onClick={clearCache}/></div>
```

Comment:
I'm wondering if the alt string should be in CamelCase but probably just a space between words. For an audio reader program for the sight-impaired, "ch" is a hard thing and cache is probably special-cased. I wonder if it will catch the special case if you make up a non-word like this, so the pronunciation may be bad i you don't make it look like its component words. Maybe it understands CamelCase, maybe not. I think just alt="clear cache" is clearer.

Reply:
Agreed, will make change and keep in mind. Alt tags should be more obviously descriptive.